src/logic/FakeTouchEvent.js

```diff
@@ -17,8 +17,8 @@ function bitCodeMouseButton(button) {
 
 export function dispatchFakeEvent(originalEvent, touch, mouseButton, type, target = touch.target) {
   const mouseEventInitProperties = {
-    clientX: touch.clientX,
-    clientY: touch.clientY,
+    clientX: touch.current?touch.current.x:touch.clientX,
+    clientY: touch.current?touch.current.y:touch.clientY,
```

Comment:
Formatting: Please put a space around the ternary operator symbols: `touch.current ? touch.current.x : touch.clientX`